src/processor/ExtensionProcessor.ts

```diff
@@ -8,7 +8,8 @@ import { ProcessableElementDefinition } from '.';
 export class ExtensionProcessor extends AbstractSDProcessor {
   static process(
     input: any,
-    fhir: fhirdefs.FHIRDefinitions
+    fhir: fhirdefs.FHIRDefinitions,
+    existingInvariants: ExportableInvariant[] = []
```

Comment:
Kind of a bummer that we need to pass this all around now -- but I don't really see a better option.  Oh well.